trace-obfuscation/src/ip_address.rs

```diff
@@ -0,0 +1,280 @@
+// Copyright 2024-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+use lazy_static::lazy_static;
```

Comment:
In this case since the types are not Sync you'll have to use OnceLock.
```suggestion
use std::sync::OnceLock
```